### PR TITLE
feat: Add Mailchimp connector

### LINKED ISSUE #152: Implement components for Mailchimp
Dear team. Thanks for the great work around this lib. I am currently trying to use additional connectors from the CDK, first one being mailchimp. It is currently not available, but I managed to write my own MailChimpConnectorProfile based on your library. Do you think it would be possible to merge it in to your next release? Or are you planning on releasing more ConnectorsProfile soon ?

Thanks

### PR DESCRIPTION
**Pull request by remitache**
Fixes #152 

This PR adds the MailChimp connector profile using api_key connection.

### CODE REVIEW

File: src/mailchimp/profile.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+*/
+import { ConnectorAuthenticationType } from '../core/connectors/connector-authentication-type';
+import { ConnectorProfileBase, ConnectorProfileProps } from '../core/connectors/connector-profile';
+import { CfnConnectorProfile } from "aws-cdk-lib/aws-appflow";
+import { Construct } from "constructs";
+import { MailChimpConnectorType } from "./type";
+
+
+export interface MailChimpConnectorProfileProps extends ConnectorProfileProps {
+    readonly apiKey: string;
```
Comment (psrebniak) [inline src/mailchimp/profile.ts:13]: apiKey should be passed as a `SecretValue`

File: src/mailchimp/source.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { MailChimpConnectorType } from "./type";
+import { IConstruct } from "constructs";
+
+export interface MailChimpSourceProps {
+    readonly profile: MailChimpConnectorProfile;
+    readonly apiVersion: string;
+    readonly object: string;
+}
+/**
+ * A class that represents a Google Analytics v4 Source
```
Comment (psrebniak) [inline src/mailchimp/source.ts:19]: * A class that represents a MailChimp v3 Source

**Review by psrebniak — CHANGES_REQUESTED** [review]
Thanks for raising the PR. 
Few minor comments, plus we need to ensure unit test coverage - could you please add one as well? You can use SalesForce tests as a good starting point

**Comment by psrebniak** [comment]
@remitache please take a look on PR I created to your branch - fixed few problems with profile etc https://github.com/remitache/cdk-appflow/pull/1

**Comment by psrebniak** [comment]
@remitache thanks for your contribution, really appreciate.

@rpawlaszek @kozub please double check and approve

**Review by rpawlaszek — APPROVED** [review]
LGTM

**Comment by psrebniak** [comment]
Related issue: #152

**Pull request merged**